src/bun_js.zig

```diff
@@ -504,5 +503,5 @@ noinline fn dumpBuildError(vm: *JSC.VirtualMachine) void {
 noinline fn failWithBuildError(vm: *JSC.VirtualMachine) noreturn {
     @setCold(true);
     dumpBuildError(vm);
-    Global.exit(1);
+    Global.exit(vm.exit_handler.exit_code);
```

Comment:
Is this right? Do any tests verify that a build error does not exit with code 0?

Reply:
I'm not sure how to trigger the errors here actually